ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/DirectionOrderMapper.java

```diff
@@ -48,6 +48,7 @@ public DirectionOrder from(CCDDirectionOrder ccdDirectionOrder, CCDOrderGenerati
 
         DirectionOrder directionOrder = builder
             .createdOn(ccdDirectionOrder.getCreatedOn())
+            .hearingCourtName(ccdDirectionOrder.getHearingCourtName())
```

Comment:
Shouldn't we have `to` mapping as well? Also missing unit tests for this

Reply:
no to as this is just to display things at frontend, but no change is made in it at front end